extract_dmd.d

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env dub
+/++
+dub.sdl:
+dependency "libdparse" version="0.7.0-beta.7"
+dependency "mustache-d" version="0.1.3"
```

Comment:
Ah yes, the `traits.dd` ones were not loaded so didn't show up in Ctrl+F.

Still I think it would be good to avoid adding a third template engine to the dlang.org build process (on top of DDoc and Diet). In fact, it would be more idiomatic to generate a .ddoc file and use a DDoc macro - but that's all pending the larger decision on whether we want to do this.